pages/place/routers/delete.js

```diff
@@ -0,0 +1,47 @@
+const { Router } = require('express');
+const bodyParser = require('body-parser');
+
+module.exports = () => {
+  const app = Router();
+
+  const _configure = (req, res, next) => {
+    req.form = {};
+    req.model = req.model || {};
+    req.session.form = req.session.form || {};
+    return next();
+  };
+
+  const _checkDelete = (req, res, next) => {
+    if (req.body.delete) {
+      req.form.delete = true;
+      return next();
+    }
+    return next(new Error('Something went wrong'));
```

Comment:
Can we make the error message here actually useful please? 

They'll likely get mapped to something generic like that for _public display_ but we'll want proper errors for the logs.